packages/editor-ui/package.json

```diff
@@ -43,6 +43,7 @@
     "@jsplumb/connector-bezier": "^5.13.2",
     "@jsplumb/core": "^5.13.2",
     "@jsplumb/util": "^5.13.2",
+    "@sendgrid/client": "^7.7.0",
```

Comment:
Why did you install SendGrid client?

Reply:
That's what we're using to fetch the templates.